tests/cloner_test.go

```diff
@@ -98,14 +98,15 @@ var _ = Describe("all clone tests", func() {
 			fmt.Fprintf(GinkgoWriter, "INFO: %s\n", sourcePvcDiskGroup)
 			Expect(err).ToNot(HaveOccurred())
 
-			By("verifying pvc content")
-			sourceMD5, err := f.GetMD5(f.Namespace, pvc, diskImagePath, 0)
+			By("Calculating the md5sum of the source data volume")
+			sourceMD5, err := f.RunCommandAndCaptureOutput(utils.PersistentVolumeClaimFromDataVolume(dataVolume), "md5sum "+diskImagePath)
 			Expect(err).ToNot(HaveOccurred())
+			fmt.Fprintf(GinkgoWriter, "INFO: MD5SUM for source is: %s\n", sourceMD5[:32])
 
 			By("Deleting verifier pod")
 			err = utils.DeleteVerifierPod(f.K8sClient, f.Namespace.Name)
```

Comment:
no need to delete `verifier` pod if execute-command Pod is created and already deleted.